### PR TITLE
Michail models new features

### PR DESCRIPTION
**Pull request by kovanostra**
Reran notebooks with new features. The xgboost one shows a deprecation warning that should be ignored (sklearn issue which appears on the last official release but not on the master). The confusion matrix for the validation confusion matrix throws an error, but there is no error on the test set. This error is strange because it does not appear in the sklearn notebook (even though conditions are exactly the same), therefore my belief is that it should be ignored. Ignored creatinine_yesterday, creatinine_before_yeasterday and all delays in the runs shown in these notebooks.

### CODE REVIEW

**Comment by Aurelie-Mutschler** [comment]
thanks @kovanostra ;)
